user-bot/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import { exec } from 'child_process';
+import { run } from './app';
+
+function init() {
+  exec('echo "checking notifications"', async (error, _stdout, stderr) => {
+    if (error) {
+      console.error(`Error: ${error.message}`);
+      return;
+    }
+
+    if (stderr) {
+      console.error(`Stderr: ${stderr}`);
+      return;
+    }
+
+    await run();
+
+    setTimeout(init, Number(process.env.DELAY_VERIFICATION || 5000));
+  });
+}
+
+console.log('Started');
+
+init();
```

Comment:
The user-bot code is out of date, I think only this index was changed

https://github.com/gibwork/gibwork-github-tips/blob/main/src/index.ts

Reply:
Done! 
sorry about that😅